cmd/poller/poller.go

```diff
@@ -106,85 +107,93 @@ func (me *Poller) Init() error {
 	options.SetPathsAndHostname(&args)
 	me.options = &args
 	me.name = args.Poller
-	logger.Info(me.prefix, "options config: %s", me.options.Config)
-
-	// use prefix for logging
-	me.prefix = "(poller) (" + me.name + ")"
 
+	fileLoggingEnabled := false
+	consoleLoggingEnabled := false
+	zeroLogLevel := logging.GetZerologLevel(me.options.LogLevel)
 	// if we are daemon, use file logging
 	if me.options.Daemon {
-		_LOG_FILE_NAME = "poller_" + me.name + ".log"
-		if err = logger.OpenFileOutput(me.options.LogPath, _LOG_FILE_NAME); err != nil {
-			return err
+		logFileName = "poller_" + me.name + ".log"
+		fileLoggingEnabled = true
+	} else {
+		consoleLoggingEnabled = true
+	}
+
+	// log handling parameters
+	// size of file before rotating
+	if s := me.params.GetChildContentS("log_max_bytes"); s != "" {
```

Comment:
seg fault since `params` is nil

Reply:
Actually i did remove these in earliar commit then saw we make references to them in read me.
I am of opinion that we don't need these configuration. Probably also doesn't fit well with our docker story.
To log initial messages, i changed the log to point to Harvest.log and then point to poller logs. That doesn't seem like right approach.
Another common problem I sense if there will be too many file descriptors open for harvest.log (default config) from each poller? Let's discuss this more in meeting.

Reply:
Addressed these concerns as discussed in meeting.